classy_config/loader/env_loader.py

```diff
@@ -0,0 +1,21 @@
+from os import environ
+from pathlib import Path
+from typing import MutableMapping
+
+from dotenv import load_dotenv
+
+
+def env_loader(filepath: Path) -> MutableMapping[str, str]:
+    """
+    Load the given *.env file and the user's environment variables.
+
+    Prioritizes system/user environment variables. Will not override any existing environment variables from the
+    system.
+
+    :param filepath: The path of the file to load from.
+    :return: The mapping of keys to value loaded from the env file and user's environment.
+    """
+    if filepath:
```

Comment:
Shouldn't this be checking if the file exists?
```suggestion
    if filepath.exists():
```

Reply:
Would this be covered by this check [here](https://github.com/fisher60/classy-config/blob/feature/dotenv-loading/classy_config/config.py#L36)? If not, maybe we should add a generic check in the loader instead of handling it per loader. 

This check was originally added since python-dotenv is already configured to load the file named `.env`, so the filepath is optional, we can likely remove the check here since filepath is always required.

Reply:
I agree with removing the check here, it can be assumed it will always exist. Good spot.